README.md

```diff
@@ -1,13 +1,55 @@
-# Pizza Bot
+# The project: Pizza Bot
 
-Replace this readme with your own information about your project. 
+The goal for this project was to create a pizzabot using javascript. We needed to gather data from the user in order to process the users order. This included username, pizza type and amount of pizzas. All of this information needed to be made visible in the DOM.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## How we built the pizzabot, the problem's we came across, what we learnt & if we had more time
 
-## The problem
+How we built it:
+1. We started by reading through the project and then doing the iteration 1, 2 3 and 4 which we found the easiest. 
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+2. Then we started to add the conditional statements. The first one where you are meant to create an if else statement; if the user inputs a pizza that is on the menu then that will be true and if not then it'll be false. The second if, else if and else statements felt understandable. 
+
+3. Then we got really stuck on the functions. I think this was because we didn't really understand how they work and what arguments to write in the brackets. As well as how to call these and then use/store the information that was created when the function was run. After a bit of help and some googaling we understood better how we were meant to use these. We also created two big functions where all the other small functions were called based on user input and if it was true or false. 
+
+4. Once we got the functions to work together we started to edit the prompts and alerts so we could view the information in the DOM. We used the document.getElementById().innerhtml to connect both the javascript sections to the html. 
+
+5. For the end result we managed to get the form functions and onsubmit attribute of the form to trigger the next function and the same for the next function. This meant that when the user inputted information and submitted with the submit button it made the next form section appear. 
```

Comment:
This worked out super well!  You should be proud! 👏

Reply:
Thank you! We did get some help with this :)